python/taichi/ui/scene.py

```diff
@@ -88,6 +88,35 @@ def set_camera(self, camera):
         """
         self.scene.set_camera(camera.ptr)
 
+    def lines(self,
+              vertices,
+              width,
+              indices=None,
+              color=(0.5, 0.5, 0.5),
+              per_vertex_color=None):
```

Comment:
Consider adding `vertex_offset`, `vertex_count`, `index_offset` and `index_count` for API symmetry as `mesh`.

Reply:
OK, working on this from now.

Reply:
I will add those details in the next version.